pkgs/development/python-modules/file_read_backwards/default.nix

```
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, mock }:
+
+buildPythonPackage rec {
+  pname = "file-read-backwards";
+  version = "2.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
```

It turns out that the project's maintainer uploads files with an incorrect name (using underscores, see [list of files in pypi](https://pypi.org/project/file-read-backwards/2.0.0/#files)).

Maybe do this:
```suggestion
    pname = builtins.replaceStrings ["-"] ["_"] pname;
    inherit version;
```